### PR TITLE
chore: Upgrade to serverless@2.x

### PR DESCRIPTION
**Pull request by badr4php**
Related to Jira: [ENG-1447]

[ENG-1447]: https://comicrelief.atlassian.net/browse/ENG-1447?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 service: serverless-starter-app
 
-frameworkVersion: ">=1.1.0 <2.0.0"
+frameworkVersion: ">=2.0.0 <3.0.0"
```
[the remark this answers is not in the document]
Comment (badr4php) [reply serverless.yml]: Done

File: serverless.yml [hunk 2]
```diff
@@ -27,7 +27,6 @@ functions:
 
 plugins:
   - serverless-webpack
-  - serverless-dotenv-plugin
```
Comment (seb-cr) [inline serverless.yml:30]: You need to add `useDotenv: true` in order to replace the functionality of `serverless-dotenv-plugin`. See the docs I linked previously.

**Comment by badr4php** [comment]
@zhibek @seb-cr can you review this branch?

**Comment by corinja** [comment]
@seb-cr @pauldolden I've just noticed this, and [this](https://github.com/comicrelief/serverless-starter-app/pull/59). I will have a look now and try to get these merged in, unless you have other views about it.

**Review by corinja — APPROVED** [review]

**Pull request merged**